src/CallWebhookJob.php

```diff
@@ -79,7 +80,7 @@ public function handle()
 
             return;
         } catch (Exception $exception) {
-            if ($exception instanceof RequestException) {
+            if ($exception instanceof TransferException || $exception instanceof RequestException) {
```

Comment:
`Class GuzzleHttp\Exception\ConnectException now extends GuzzleHttp\Exception\TransferException instead of GuzzleHttp\Exception\RequestException.`

This is a part of the upgrade guide. I've added a new exception here but it sounds like it is not working. Never worked with standalone packages supporting a range of versions so idk what to do next.

Reply:
@freekmurze can you take a look here? 😇